typescript/infra/scripts/agent-utils.ts

```diff
@@ -90,6 +93,12 @@ export function getArgs() {
     .alias('e', 'environment');
 }
 
+export function withBalanceThresholdConfig<T>(args: Argv<T>) {
```

Comment:
starting to think these withXYZ helpers should be in their own `*-utils.ts` file, since these have nothing to do with agents